packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -60,7 +60,8 @@ management functionality with remote management functionality.
 
 %package cockpit
 BuildArch:  noarch
-Requires:   cockpit
+Requires: cockpit
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
```

Comment:
Isn't `%name` sufficient?

Reply:
honestly? I just copied what's in the `doc` package ;)

Reply:
and the template:

https://github.com/theforeman/foreman-packaging/blob/8363a28d884d5ff4361bb9bc3e1eaf6d8729cfc4/gem2rpm/scl.spec.erb#L73